### PR TITLE
bears: Add DistributionRequirements

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

File: bears/c_languages/GNUIndentBear.py [hunk 1]
```diff
@@ -18,6 +20,7 @@ class GNUIndentBear:
     """
 
     LANGUAGES = {"C", "C++"}
+    REQUIREMENTS = {DistributionRequirement('indent')}
```
Comment (sils1297) [inline bears/c_languages/GNUIndentBear.py]: no manager? :/

File: bears/go/GoErrCheckBear.py [hunk 2]
```diff
@@ -16,6 +17,7 @@ class GoErrCheckBear:
     For more information on the analysis visit <https://github.com/kisielk/errcheck>.
     """
     LANGUAGES = {"Go"}
+    REQUIREMENTS = {GoRequirement('github.com/kisielk/errcheck', '', '-u')}
```
Comment (sils1297) [inline bears/go/GoErrCheckBear.py]: can you use keyword arguments to make it a bit more obvious what is what?

**Comment by Asnelchristian** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/637/commits/b62ebad9c455b160d0c8b7c083b0b10912305085 https://github.com/coala-analyzer/coala-bears/pull/637/commits/ad54024a022a769edb4c840f6b568ad5586ef05e

**Comment by Asnelchristian** [comment]
i think you need a rebase

**Comment by Adrianzatreanu** [comment]
reack f2ca469 bd0e016

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**